nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java

```diff
@@ -219,6 +247,16 @@ public void checkComponentRemovedWhenAssetRemoved() throws IOException {
     assertThat(component, is(equalTo(null)));
   }
 
+  @Test
+  public void failRetrieveConaninfoWhenWrongHashInConanmanifest() throws Exception {
+    CloseableHttpResponse response = proxyClient.get(LIB_WITH_WRONG_CONANINFO_HASH_DOWNLOAD_URLS_PATH);
+    response.close();
+    response = proxyClient.get(LIB_WITH_WRONG_CONANINFO_HASH_CONANMANIFEST_PATH);
+    response.close();
+    assertThat(status(proxyClient.get(LIB_WITH_WRONG_CONANINFO_HASH_CONANINFO_PATH)), is(HttpStatus.NOT_FOUND));
```

Comment:
use try with resources or try, finally

Reply:
Same as previous.

Reply:
As we discussed I have added some comments to make test flow more clear.
e7d5a0b1c7948d6a8d7708f9a90b2388dee97648